5/map_and_set/script.js

```diff
@@ -0,0 +1,51 @@
+//TODO create a map object
+
+const someMap = new Map([
+    ["1", ['serg', 'borg']], [2, ['borg', 'serg']]
+]);
+
+console.log(someMap);
+
+//? super random
+// for (let entry of someMap.entries()) {
+//     alert(`key ${entry[0]} with value ${entry[1]}`);
+// }
+
+//? better than above
+for (let key of someMap.keys()) {
+    // alert(`key ${key} with value ${someMap.get(key)}`);
+}
+
+
+//TODO Filter unique array members
+
+function unique(arr) {
+    const set = new Set(arr);
+    // console.log(set);
+    return Array.from(set); //? need to convert Set to Array
+}
+
+let values = ["Hare", "Krishna", "Hare", "Krishna",
+    "Krishna", "Krishna", "Hare", "Hare", ":-O"
+];
+
+// alert(unique(values));
+
+//TODO Filter anagrams
+
+
+
+//TODO Iterable keys
+
+let map = new Map();
+
+map.set("name", "John");
+
+let keys = Array.from(map.keys());
+
+keys.push("more");
+
+console.log(keys); // name, more
+console.log(map); //? can we update our map with a new key?
```

Comment:
map.set('whateverKey');

if you add two things like *map.set('whateverKey', 123)* it will set as a pair of key-value.

or i probably misunderstood this question? or was it a question to us at all?

Reply:
I think I was confused here a bit. To be honest I don't even remember what was I trying to do....